### PR TITLE
feat!: display dsp release number (DEV-420)

### PR DESCRIPTION
**Pull request by kilchenmann**
resolves DEV-420

This PR contains a breaking change because it needs an updated version of the configuration.

### CODE REVIEW

**Comment by kilchenmann** [comment]
@mdelez I've attached the screenshots of the app in different environments directly to the jira-task description.

File: src/app/main/header/header.component.html [hunk 1]
```diff
@@ -4,7 +4,18 @@
...
-            <p class="mat-caption right app-version">{{appVersion}}</p>
+            <p class="mat-caption right app-version">
+                <span *ngIf="dsp.production">{{dsp.release}}</span>
+                <span *ngIf="!dsp.production" class="development badge">
+                    <span class="environment">{{dsp.environment}}</span>
+                    <span class="release {{dsp.color}}">{{dsp.release}}</span>
+                </span>
+            </p>
+
+            <!-- environment info-text {{dsp.color}}" [class.strong]="!dsp.production"> -->
```
Comment (mdelez) [inline src/app/main/header/header.component.html:15]: can this be removed?

File: src/app/main/header/header.component.ts [hunk 2]
```diff
@@ -4,13 +4,15 @@ import { MatIconRegistry } from '@angular/material/icon';
...
+import { AppInitService } from 'src/app/app-init.service';
 import { DialogComponent } from 'src/app/main/dialog/dialog.component';
 import { ComponentCommunicationEventService, Events } from 'src/app/main/services/component-communication-event.service';
 import { SearchParams } from 'src/app/workspace/results/list-view/list-view.component';
+import { DspConfig } from '../declarations/dsp-config';
 import { NotificationService } from '../services/notification.service';
 import { SessionService } from '../services/session.service';
 
-const { version: appVersion } = require('../../../../package.json');
+// const { version: appVersion } = require('../../../../package.json');
```
Comment (mdelez) [inline src/app/main/header/header.component.ts:15]: this can also be removed I'm guessing

**Review by mdelez — CHANGES_REQUESTED** [review]
Works well for me! Just some things to clean up :)

File: src/app/main/header/header.component.html (hunk 1 above)
Comment (kilchenmann) [reply src/app/main/header/header.component.html]: done in 65607173

File: src/app/main/header/header.component.ts (hunk 2 above)
Comment (kilchenmann) [reply src/app/main/header/header.component.ts]: done in 65607173

**Review by mdelez — APPROVED** [review]
Thank you for this, it's super helpful!

**Pull request merged**